lib/si-data-pg/src/lib.rs

```diff
@@ -66,7 +66,7 @@ const TEST_QUERY: &str = "SELECT 1";
 // We set `CLIENT_MIN_MESSAGES` to `ERROR` to silence the possible warning from `ROLLBACK` about
 // there not being any open transaction. We immediately set it back to the default value
 // (`WARNING`) after so we're not hiding any "real" warnings that might happen.
-const CONNECTION_RECYCLING_METHOD: &str = r#"
+pub const CONNECTION_RECYCLING_METHOD: &str = r#"
```

Comment:
Approved, but why is this now pub?

Reply:
I think this actually works well though because you're implying that an external user could use it if they wanted to

Reply:
Fixed